unittest/unit/update-bcd.ts

```diff
@@ -648,6 +710,8 @@ describe('BCD updater', () => {
         chrome: [{version_added: '0> ≤83', version_removed: '85'}]
       },
       'api.ExperimentalInterface': {chrome: [{version_added: '0> ≤83'}]},
+      'api.UnflaggedInterface': {chrome: [{version_added: '0> ≤84'}]},
```

Comment:
Yeah, this `0>` is kind of weird :)

Reply:
0> should not be appearing in output values, hmm...

Reply:
GitHub's "review" interface is obscuring the context a bit, so it may not be clear that this particular test is for the `inferSupportStatements` function. That may include `0>` in its return value, but it may not be what you mean by "output," at least not from the perspective of someone using  the `update-bcd` script.

The `update` function, on the other hand, contains a handful of statements intended to remove `0>`. This patch exercises [this replacement](https://github.com/foolip/mdn-bcd-collector/pull/2297/files#diff-4cdb8f3a80ed4479be24afc5f4ec7f4c0f29c956731945d53a34691a97f0ede1R362-R365); disabling that code causes the test for the `update` function to fail (specifically for the `UnflaggedInterface` and `UnprefixedInterface` added here).